apps/cowswap-frontend/src/modules/swap/services/ethFlow/index.ts

```diff
@@ -66,6 +67,12 @@ export async function ethFlow(
   )
 
   try {
+    // Do not proceed if fee is expired
+    if (isQuoteExpired(fee.expirationDate)) {
+      reportPlaceOrderWithExpiredQuote({ ...orderParamsOriginal, fee })
+      throw new Error('Quote expired. Please refresh.')
```

Comment:
Break the flow.
The error message is what will be displayed in the error modal

Reply:
But will not happen normally, right? as you disabled the button wile is updating

Reply:
No it shouldn't. But can still happen if the user is on the confirmation screen and goes away for 2h.